crew

```diff
@@ -444,21 +444,18 @@ def unpack (meta)
   extract_dir = "#{meta[:filename]}.dir"
   target_dir = nil
   Dir.chdir CREW_BREW_DIR do
-    puts "Unpacking archive, this may take awhile..."
     Dir.mkdir("#{extract_dir}") unless Dir.exist?("#{extract_dir}")
-    case File.extname meta[:filename]
-    when '.zip'
-      if @opt_verbose then
-        system "unzip", "-v", "-d", "#{extract_dir}", meta[:filename]
-      else
-        system "unzip", "-qq", "-d", "#{extract_dir}", meta[:filename]
-      end
+    case File.basename meta[:filename]
+    when /\.zip$/i
+      puts "Unpacking archive using zip, this may take awhile..."
```

Comment:
Extracting archive using unzip?

Reply:
I basically kept the original calls to the `#system` method here; but if you want to change anything else while we're at it, let me know.

Reply:
Yea, I clearly did not get this at first :) - changed it with the latest version...